tests/config/test_config.py

```diff
@@ -164,6 +166,16 @@ def test_default_config(self):
         conf = config.ConfigManager(None, None)
         conf._validate_all()
 
+    def test_default_key_config(self):
+        """Test validating of the default key config."""
+        # We import qutebrowser.app so the cmdutils.register decorators run.
+        import qutebrowser.app
+        conf = keyconf.KeyConfigParser(None, None)
+        runner = runners.CommandRunner(win_id=0)
+        for sectname in configdata.KEY_DATA:
```

Comment:
This could be parametrized, if you would like to take the opportunity to write your first parametrized test. :smile: 

Otherwise it looks fine to me! :+1: